component/vshn_postgres.jsonnet

```diff
@@ -119,6 +118,14 @@ local composition =
                               sgInstanceProfile: '',
                               configurations: {
                                 sgPostgresConfig: '',
+                                backups:
+                                  [
+                                    {
+                                      sgObjectStorage: '',
+                                      cronSchedule: '0 22 * * *',
```

Comment:
Why do you expose the Schedule in the claim meanwhile you hardcode the value in the composition? Am I missing something?

Reply:
will be fixed, thanks for noticing